README.md

```diff
@@ -8,39 +8,119 @@ HTTP sink connector reads records from data streaming and generates an HTTP requ
 
 > Supports HTTP/1.0, HTTP/1.1, HTTP/2.0 protocols.
 
+### Configuration
 
-See [docs][1] here.
+HTTP Sink is configured using a YAML file:
 
-### Configuration
+```yaml
+# config-example.yaml
+apiVersion: 0.1.0
+meta:
+  version: 0.1.0
+  name: my-http-sink
+  type: http-sink
+  topic: http-sink-topic
+  secrets:
+    - name: HTTP_TOKEN
+http:
+  endpoint: "http://127.0.0.1/post"
+  headers:
+    - "Authorization: token ${{ secrets.HTTP_TOKEN }}"
+    - "Cache-Control: no-cache"
+```
 
 | Option       | default                    | type            | description                                       |
 | :------------| :--------------------------| :-------------- | :-------------------------------------------------|
 | method       | POST                       | String          | POST, PUT                                         |
 | endpoint     | -                          | String          | HTTP URL endpoint                                 |
 | headers      | -                          | Array\<String\> | Request header(s) "Key:Value" pairs               |
-| user-agent   | "fluvio/http-sink 0.1.0"   | String          | Request user-agent                                |
+| user-agent   | `fluvio/http-sink 0.1.0`   | String          | Request user-agent                                |
 
 > By default HTTP headers will use `Content-Type: text/html` unless anothed value
 > is provided to the Headers configuration.
 
-### Usage Example
+### Usage
+
+Login to your Fluvio Cloud Account via Fluvio CLI
+
+```bash
+fluvio cloud login --use-oauth2
+```
 
-This is an example of simple connector config file:
+Then configure the HTTP Request to be sent using a YAML file following the
+connector configuration schema. The following configuration will send a POST
+HTTP request to `http://httpbin.org/post`.
 
 ```yaml
-# config-example.yaml
+# config.yaml
+apiVersion: 0.1.0
 meta:
   version: 0.1.0
-  name: my-http-sink
+  name: httpbin
   type: http-sink
-  topic: http-sink-topic
+  topic: httpbin-send-post
+
 http:
-  endpoint: "http://127.0.0.1/post"
-  headers:
-    - "Authorization: token MySecretToken"
-    - "Cache-Control: no-cache"
+  endpoint: http://httpbin.org/post
+  interval: 3s
+```
+
+Finally create your connector by running:
+
+```bash
+fluvio cloud connector create --config ./config.yaml
+```
+
+> You can see active connectors by running the following command:
+>
+> ```bash
+> fluvio cloud connector list
+> ```
+
+Check connector logs by running
+
+```bash
+fluvio cloud connector logs <NAME>
```

Comment:
I know this is a closed `pr`, but we should change <NAME> to `fluvio cloud connector logs httpbin` as the readme focuses on this particular use case.

Reply:
Of course!